packages/dnb-eufemia/src/extensions/forms/Field/PostalCodeAndCity.tsx

```diff
@@ -3,8 +3,10 @@ import classnames from 'classnames'
 import SharedContext from '../../../shared/Context'
 import FieldBlock, { Props as FieldBlockProps } from '../FieldBlock'
 import StringComponent, { Props as StringComponentProps } from './String'
+import { FieldHelpProps } from '../types'
 
-export type Props = Omit<FieldBlockProps, 'children'> &
+export type Props = FieldHelpProps &
```

Comment:
```suggestion
export type Props = 
```

I think, `help` should be a part of `StringComponentProps`, is it not?

Reply:
Correct, `help` is part of `StringComponentProps`.
But this component `PostalCodeAndCity` doesn't extend/inherit props from `StringComponentProps`.
I could of course do `Pick<StringComponentProps, 'help'>` instead of `FieldHelpProps`, but I think `FieldHelpProps` is "better" or more declarative.

Reply:
But because of the existing type in `PostalCodeAndCity`: `Record<'postalCode' | 'city', StringComponentProps>`.
Maybe I don't need to add a specific help prop to the component, as I believe they could do the following:
´´´
      <Field.PostalCodeAndCity
        postalCode={{
          help: {
            title: 'Help is available',
            contents:
              'Here is what a team can do for you. . . . It allows you to help others do their best.',
          },
        }}
        city={{
          help: {
            title: 'Help is available',
            contents:
              'Here is what a team can do for you. . . . It allows you to help others do their best.',
          },
        }}
      />
´´´

Maybe I should rather remove the code of this PR where I add the prop `help`, and rather just update the example to showcase the example above?

Reply:
haha, yeah. Let's go for one help button only here. So we need to set the prop on the component directly and not inside `city` or `postalCode`.